lib/importers/google/satisfaction/call_record.rb

```diff
@@ -3,9 +3,10 @@ module Importers
   module Google
     module Satisfaction
       class CallRecord
+        class NotImplementedError < StandardError; end
```

Comment:
You're redefining an existing error class here. `NotImplementedError` is in the stdlib.